projects/plugins/jetpack/class-jetpack-stats-report-upsells.php

```diff
@@ -0,0 +1,214 @@
+<?php
+/**
+ * Adds a section with upsell CTAs to the Status Report page
+ *
+ * @package jetpack
+ */
+
+use Automattic\Jetpack\Jetpack_CRM_Data;
+
+/**
+ * Class that adds a new section to the Stats Report page
+ */
+class Jetpack_Stats_Report_Upsells {
```

Comment:
I wonder if we could use the same vocabulary as in the other areas of the dashboard, and use the word "feature upgrade nudges" to remain consistent across the codebase.

Reply:
ok! I wasn't sure how to call this

Reply:
done